src/transpilation/resolve.ts

```diff
@@ -93,6 +98,60 @@ class ResolutionContext {
         }
     }
 
+    private resolveDependencyPathsWithPlugins(required: ProcessedFile, dependency: string) {
+        const requiredFromLuaFile = required.fileName.endsWith(".lua");
+        const dependencyPath = requiredFromLuaFile ? luaRequireToPath(dependency) : dependency;
+
+        for (const plugin of this.plugins) {
+            if (plugin.moduleResolution != null) {
+                const pluginResolvedPath = plugin.moduleResolution(
+                    dependency,
+                    dependencyPath,
```

Comment:
this parameter is the requiringFile, shouldn't this be `required.fileName`?

Reply:
I missed that, which make sense why I ran into the issues I did, I'll fix that ty